gateway-service-impl/src/test/resources/test-requests/explore/simple-aggregations.json

```diff
@@ -64,8 +64,8 @@
           {
             "literal": {
               "value": {
-                "valueType": "LONG",
-                "long": "2"
+                "valueType": "STRING",
```

Comment:
Should be the same as prior, right? If requires, we need to add a new request test.

Reply:
Yes, now, you can revert back.